bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/handler/WemoDimmerHandler.java

```diff
@@ -197,9 +204,11 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                             value = "0";
                             argument = "BinaryState";
                             setBinaryState(action, argument, "0");
-                        } else if (this.stateMap.get("BinaryState").equals("0")) {
-                            argument = "BinaryState";
-                            setBinaryState(action, argument, "1");
+                        } else if (binaryState != null) {
+                            if (binaryState.equals("0")) {
```

Comment:
You can skip these null checks by calling the method on non-null objects `"0".equals(binaryState)`

Reply:
Done